canvas_modules/common-canvas/src/common-properties/components/field-picker/field-picker.jsx

```diff
@@ -436,6 +437,7 @@ export default class FieldPicker extends React.Component {
 				filterKeyword={this.state.filterText}
 				scrollKey="field-picker"
 				rows={-1}
+				control={fieldPickerTitle}
```

Comment:
How come the title is being passed into `control`? Should `control` be renamed to something else instead so its not misleading?